packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1,6 +1,6 @@
 import { axisRight as d3AxisRight, axisBottom as d3AxisBottom, axisLeft as d3AxisLeft, Axis as D3Axis } from 'd3-axis';
 import { max as d3Max, min as d3Min } from 'd3-array';
-import { scaleLinear as d3ScaleLinear, scaleTime as d3ScaleTime, scaleBand as d3ScaleBand } from 'd3-scale';
+import { scaleLinear as d3ScaleLinear, scaleBand as d3ScaleBand, scaleUtc as d3ScaleUtc } from 'd3-scale';
```

Comment:
>scaleBand [](http://example.com/codeflow?start=39&length=9)

no snapshots updated due to this change?

Reply:
There are changes. I tried reverting it and then ran the `test:tz` script. Some snapshot tests started failing in timezones other than UTC.